vocabularies/Offline.xml

```diff
@@ -0,0 +1,57 @@
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="com.sap.vocabularies.Offline.v1" Alias="Offline">
+      <Annotation Term="Core.Description">
+        <String>Terms for annotating Offline OData</String>
+      </Annotation>
+      <Annotation Term="Core.LongDescription">
+        <String>
+Offline OData is powerful solution for data sycnchronization between back-end and mobile devices.
+It is built on top of in-house technology which has been designed and used in database synchronizing for over 20 years.
+Plese refer to the Offline OData product home for details:
+https://help.sap.com/doc/f53c64b93e5140918d676b927a3cd65b/Cloud/en-US/docs-en/guides/features/offline/overview.html
+        </String>
+     </Annotation>
+      <Annotation Term="Core.Description" Qualifier="Published">
+        <String>2022-10-27 © Copyright 2022 SAP SE. All rights reserved</String>
+      </Annotation>
+      <Annotation Term="Core.Links">
+        <Collection>
+          <Record>
+            <PropertyValue Property="rel" String="latest-version" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/Offline.xml" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="alternate" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/Offline.json" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="describedby" />
+            <PropertyValue Property="href" String="https://github.com/sap/odata-vocabularies/blob/main/vocabularies/Offline.md" />
+          </Record>
+        </Collection>
+      </Annotation>
+
+      <Term Name="ClientOnly" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntityType EntitySet EnumType ComplexType TypeDefinition">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated model element exists only on client device"/>
+         <Annotation Term="Core.LongDescription">
+           <String>Occasionally a customer will want to store additional “client-only”
+           entities in the same DB. The easiest way to accommodate this is for the client 
+           to extend the backend-returned $metadata (upon which the local persistence is 
+           based) and just to mark some entities as client-only so that the client won’t
+           attempt to upload any changes that are made to them locally.
+           The service as implemented on the server is described by the CSDL document with
+           all elements removed that are tagged as `Offline.ClientOnly`.</String>
```

Comment:
```suggestion
           The service as implemented on the server is described by the CSDL document if
           all elements tagged as `Offline.ClientOnly` are removed together will all subsequently dangling references.</String>
```
Your example shows this addition is necessary: The `SalesOrderItem` entity set is not tagged, but must be removed together with the `SalesOrderItem` entity type.

Reply:
That is true, thanks very much!